packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -13,7 +13,7 @@
                 'is-hidden--noJS',
                 $style['c-nav-trigger'],
                 $style['c-nav-toggle'],
-                { [$style['c-nav-toggle--altColour']]: isAltColour },
+                { [$style['c-nav-toggle--altColour']]: isAltColour || (headerBackgroundTheme === 'highlight' && navIsOpen) },
```

Comment:
I thought it was better to add the additional logic here rather than extending `isAltColour` to support the unique case of:
1. Icon is the nav toggle
2. The theme is `highlight`
3. The mobile nav is open